routes/weather.ts

```diff
@@ -260,10 +254,13 @@ export const getWateringData = async function( req: express.Request, res: expres
 	// Continue with the weather request
```

Comment:
We should be able to remove `location = coordinates` line if next review comment holds true.